releasenotes/notes/cws-7-54-bb2551a1d930853b.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    CWS: Add support ``write`` event on Windows.
+  - |
+    CWS: Add support for ``rename`` event on Windows.
+  - |
+    CWS: Add ``chdir``, ``mount`` and ``umount`` in ``ptracer`` mode.
+  - |
+    CWS: Add APM spans in ``ptracer`` mode.
+  - |
+    CWS: Add support for file hashing in ``ptracer`` mode.
+enhancements:
+  - |
+    CWS: Always snapshot mmaped files in activity dumps. This allows easier mapping of workload to loaded shared libraries.
+  - |
+    CWS: Normalize Windows filenames before going through glob matching.
+  - |
+    CWS: Add ``pattern -> glob`` conversion on Windows paths.
+  - |
+    CWS: Support non seccomp mode in ``ptracer`` mode.
+other:
+  - |
+    CWS: Allow ``fim_enabled`` to explicitly be set to false to enable process monitoring only.
```

Comment:
only on windows sadly (because it's not possible to do this on linux because of the helm chart)